fern/docs-config/definition/docs.yml

```diff
@@ -28,6 +29,17 @@ types:
       path: string
       href: string
 
+  DocsTypographyConfig:
+    properties:
+      headingsFont: optional<FontConfig>
+      bodyFont: optional<FontConfig>
+      codeFont: optional<FontConfig>
+
+  FontConfig:
+    properties:
+      name: string
```

Comment:
Do we need the user to supply a name? Or is path ok?

Reply:
The font name is used to declare the name of the font in the eventually generated CSS. We could assign IDs, but it may not be helpful for debugging the front end.

Reply:
Maybe we can just default to the filename or something like that? Always nice to minimize config

Reply:
I see. That's also possible. 👍🏽